### PR TITLE
Remove special report special url

### PR DESCRIPTION
**Pull request by TheoLeanse**
@matthew-andrews @i-like-robots 

B/C of #1066

### CODE REVIEW

File: test/smoke.js [hunk 1]
```diff
@@ -8,14 +8,15 @@ module.exports = [{
 		'/special-reports/MDEzZjdiOTAtYTFhNC00NzBjLTnotareport': 404,
```
Comment (matthew-andrews) [inline test/smoke.js]: 💥

Comment (TheoLeanse) [inline test/smoke.js]: ahhhh

File: server/init.js [hunk 2]
```diff
@@ -30,16 +30,15 @@ var app = module.exports = express({
...
 // Special Reports
-app.get(`/stream/specialReportsId/:thingId`, (req, res) => {
-	res.redirect(`/special-reports/${req.params.thingId}`);
+app.get('/special-reports/:thingId', (req, res) => {
+	res.redirect(`/stream/specialReportsId/${req.params.thingId}`);
 });
 
 app.get('/special-reports.?:extension(rss|json)?', require('./controllers/special-reports'));
-app.get('/special-reports/:thingId.?:extension(rss|json)?', require('./controllers/stream'));
 app.get('/special-reports/:specialReportId/special-reports.?:extension(rss|json)?', require('./controllers/related/special-reports'));
```
Comment (matthew-andrews) [inline server/init.js]: specialReport *s* Id

File: server/init.js [hunk 3]
```diff
@@ -30,16 +30,15 @@ var app = module.exports = express({
...
 app.get('/special-reports/:specialReportId/:relatedTaxonomy.?:extension(rss|json)?', require('./controllers/related/tag'));
```
Comment (matthew-andrews) [inline server/init.js]: ^^ specialReport *s* Id 😁

File: test/smoke.js [hunk 4]
```diff
@@ -5,17 +5,18 @@ module.exports = [{
...
-		'/special-reports/MDEzZjdiOTAtYTFhNC00NzBjLTk2ZWMtMWFlOTZmZTM5Yzkz-U3BlY2lhbFJlcG9ydHM=.rss': 200,
+		'/special-reports/MDEzZjdiOTAtYTFhNC00NzBjLTk2ZWMtMWFlOTZmZTM5Yzkz-U3BlY2lhbFJlcG9ydHM=': 302,
+		'/special-reports/MDEzZjdiOTAtYTFhNC00NzBjLTk2ZWMtMWFlOTZmZTM5Yzkz-U3BlY2lhbFJlcG9ydHM=.rss': 302,
 		'/special-reports/MDEzZjdiOTAtYTFhNC00NzBjLTk2ZWMtMWFlOTZmZTM5Yzkz-U3BlY2lhbFJlcG9ydHM=/special-reports': 200,
 		'/special-reports/MDEzZjdiOTAtYTFhNC00NzBjLTk2ZWMtMWFlOTZmZTM5Yzkz-U3BlY2lhbFJlcG9ydHM=/people': 200,
 		'/news-feed': 200,
 		'/news-feed.rss': 200,
 		'/stream/peopleId/ZjdiZDJlODYtNTg1My00NTNmLTgwZTUtNjkxZTYyNTkwY2Fi-UE4=': 200,
 		'/stream/peopleId/ZjdiZDJlODYtNTg1My00NTNmLTgwZTUtNjkxZTYyNTkwY2Fi-UE4=.rss': 200,
+		'/stream/specialReportId/MDEzZjdiOTAtYTFhNC00NzBjLTk2ZWMtMWFlOTZmZTM5Yzkz-U3BlY2lhbFJlcG9ydHM=': 200,
```
Comment (matthew-andrews) [inline test/smoke.js]: specialReport **s** Id

File: test/smoke.js [hunk 5]
```diff
@@ -5,17 +5,18 @@ module.exports = [{
 	urls: {
 		// test error pages first to make sure they don't crash the app
 		'/stream/topicsId/Ng==-VG9waWNznotapage': 404,
-		'/special-reports/MDEzZjdiOTAtYTFhNC00NzBjLTnotareport': 404,
+		'/stream/specialReportId/MDEzZjdiOTAtYTFhNC00NzBjLTnotareport': 404,
```
Comment (matthew-andrews) [inline test/smoke.js]: also missing an s after t

Comment (TheoLeanse) [inline test/smoke.js]: 😢 

thanks matt

Comment (matthew-andrews) [inline test/smoke.js]: 😇

**Pull request merged**